ros_speech_recognition/scripts/parrot_node.py

```diff
@@ -20,6 +20,7 @@ def __init__(self):
             blocking=True,
             sound_action=tts_action_name,
         )
+        self.conf_thresh = rospy.get_param('~confidence_threshold','sound_play')
```

Comment:
```suggestion
        self.conf_thresh = rospy.get_param('~confidence_threshold', 0.8)
```
`~confidence_threshold` may require float.